packages/scripts/utils/config.js

```diff
@@ -72,6 +72,15 @@ const hasWebpackConfig = () =>
 	hasProjectFile( 'webpack.config.js' ) ||
 	hasProjectFile( 'webpack.config.babel.js' );
 
+const hasPostCssConfig = () =>
```

Comment:
Nit: I think it should have CSS capitalized according to code style guidelines:

```suggestion
const hasPostCSSConfig = () =>
```

But more importantly, can we include link to the documentation that lists all options for naming config files?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/22735/commits/58a8d69afcc30b570e9865104cca7af81204c2f4.